engines/dfc_provider/app/services/catalog_item_builder.rb

```diff
@@ -15,6 +15,7 @@ def self.apply_stock(item, variant)
     if limit.to_i.negative?
```

Comment:
Just looking at the above, I note that Spree::Variant should take care of this: https://github.com/openfoodfoundation/openfoodnetwork/blob/bf62d16ca5a1f9bbf6d0912d351335801b39872f/app/models/spree/variant.rb#L103

But if maybe that didn't work, maybe we can expose `Spree::Variant#create_stock_items` as a public method and call that, instead of re-defining the same logic here.

I was going to suggest we spec for it too, but it looks like it's already implicitly covered in the test by `variant.on_hand`